lib/ansible/modules/remote_management/dellemc/idrac/dellemc_idrac_server_config_profile.py

```diff
@@ -0,0 +1,304 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+#
+# Dell EMC OpenManage Ansible Modules
+# Version 1.3
+# Copyright (C) 2019 Dell Inc.
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+# All rights reserved. Dell, EMC, and other trademarks are trademarks of Dell Inc. or its subsidiaries.
+# Other trademarks may be trademarks of their respective owners.
+#
+
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: dellemc_idrac_server_config_profile
```

Comment:
Drop `dellecm_` prefix

Reply:
@jamescassell  Thank you for your comments. This is under dellemc/idrac/ , also there is a previously contributed module in this path

Reply:
@jamescassell Done the renaming part